510_Deployment/60_file_descriptors.asciidoc

```diff
@@ -1,21 +1,14 @@
 
-=== File Descriptors and MMap 
+=== 文件描述符和 MMap
 
-Lucene uses a _very_ large number of files. ((("deployment", "file descriptors and MMap"))) At the same time, Elasticsearch
-uses a large number of sockets to communicate between nodes and HTTP clients.
-All of this requires available file descriptors.((("file descriptors")))
+Lucene 使用 _非常_ 大量的文件。((("deployment", "file descriptors and MMap"))) 同时，Elasticsearch 使用了大量的套接字在节点和 HTTP 客户端之间进行通信。
```

Comment:
Lucene 会用到大量文件。同时，Elasticsearch 在节点和 HTTP 客户端之间进行通信也使用了大量的套接字。
所有这一切都需要可用的文件描述符。    @medcl  这样怎么样

2016-06-01 15:45 GMT+08:00 Medcl <notifications@github.com>:

> In 510_Deployment/60_file_descriptors.asciidoc
> <https://github.com/elasticsearch-cn/elasticsearch-definitive-guide/pull/85#discussion_r65315673>
> :
>
> >
> > -Lucene uses a _very_ large number of files. ((("deployment", "file descriptors and MMap"))) At the same time, Elasticsearch
> > -uses a large number of sockets to communicate between nodes and HTTP clients.
> > -All of this requires available file descriptors.((("file descriptors")))
> > +Lucene 使用 _非常_ 大量的文件。((("deployment", "file descriptors and MMap"))) 同时，Elasticsearch 使用了大量的套接字在节点和 HTTP 客户端之间进行通信。
>
> 数目用大来描述 感觉有点怪。
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/elasticsearch-cn/elasticsearch-definitive-guide/pull/85/files/5bf1792286a94972daf0b726d013140c7ddd5fa9#r65315673>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AEuUdTAcMeWKYItT8GeoyvVG53deL_Hzks5qHTiTgaJpZM4IGKu_>
> .
>